example/app/config.yml

```diff
@@ -22,7 +22,7 @@ default:
     instanceType: t2.small
     desiredInstances: 1
     availabilityZone: us-east-1a
-    imageId: ami-a7a242da
+    amiid: ami-a7a242da
```

Comment:
how was this working before?

Reply:
I just deployed using master and it used a different ami: `amzn-ami-2017.09.c-amazon-ecs-optimized (ami-6944c513)`

<img width="499" alt="screen shot 2018-04-17 at 1 13 30 pm" src="https://user-images.githubusercontent.com/15016780/38885516-3670634c-4241-11e8-9d97-0c8029ebc90e.png">